### ISSUE TITLE
Closing copied confirmation tooltip

### ISSUE THREAD

**Issue by arrenv** [JoinColony/colonyCDapp#2142]
Teamwork task: [Closing copied confirmation tooltip](https://tw.auroracreation.com/#/tasks/15861671)

## Description

As discovered during user interviews, the behavior of the copied confirmation tooltip is frustrating to users as it stays too long for the user and does not behave in a way they would expect or want.

Based on the various observed ways in which users interacted with this tooltip we would like to have 3 different was to close the tooltip including reducing the time the tooltip appears, allow clicking on the tooltip to close, and to close it on mouse leave.

### Changes
- Reduce the time the tooltip appears to `2 secs`.
- Have the tooltip `close on click`.
- Have the tooltip close on `mouseleave`.

### Both tooltips showing
![image](https://github.com/JoinColony/colonyCDapp/assets/33682027/d0539c1c-c65e-4f0c-bab7-7c81f69d6e06)

### Tooltip staying to long
![image](https://github.com/JoinColony/colonyCDapp/assets/33682027/4f84b3a6-50a0-442f-9f28-705d45b6e800)

**Comment by AdrianDudko** [comment]
@joanna-pagepro @CzarekDryl please estimate this change before starting working on this Issue.

**Comment by AdrianDudko** [comment]
@arrenv estimate: 4h

**Comment by arrenv** [comment]
Closing this as no longer relevant.
